common/src/main/java/com/sequenceiq/cloudbreak/auth/crn/Crn.java

```diff
@@ -138,10 +138,6 @@ public enum Service {
         NAVOPT("navopt", NON_ADMIN_SERVICE),
         DBUS("dbus", NON_ADMIN_SERVICE),
         AUTOSCALE("autoscale", NON_ADMIN_SERVICE),
-        /**
-         * @deprecated {@link #CLOUDBREAK} was replaced by {@link #DATAHUB} and is kept here for backward compatibility reasons (e.g., dynamodb serialized CRNs).
-         */
-        @Deprecated
```

Comment:
It is still kinda deprecated, mostly reserved for custom internal actors (in CB-to-CB service calls) and the REST audit event source. :smile: Would it perhaps make sense to keep the deprecated javadoc & annotation and amend the javadoc text with the permitted usages I mentioned above?

Reply:
Yes, but by the audit event source for REST will be used actively in the future and it seems to be logically correct to have a separate type for the Cloudbreak/Core service because it is more than the `Datahub`apis